src/interpreter.rs

```diff
@@ -982,7 +1015,8 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
     }
 
     fn make_aggregate_repr<V>(&self, variant_fields: V) -> Repr
-        where V: IntoIterator, V::Item: IntoIterator<Item = ty::Ty<'tcx>>
+        where V: IntoIterator,
+              V::Item: IntoIterator<Item = ty::Ty<'tcx>>
```

Comment:
Generally not a fan of alignment, but I'm not sure how else to nicely indent where clauses on multiple lines, so whatever.